src/Workspaces/Core/Portable/Shared/Extensions/ICodeDefinitionFactoryExtensions.cs

```diff
@@ -350,7 +350,40 @@ private static bool TryGetValue(IDictionary<string, ISymbol> dictionary, string
                 name: overriddenEvent.Name);
         }
 
-        public static async Task<IMethodSymbol> OverrideMethodAsync(
+        public static async Task<ISymbol> OverrideAsync(
```

Comment:
Why isn't this just in the generator itself? Seems like a helpful forwarding function.